release notes/v0.32.0.md

```diff
@@ -0,0 +1,107 @@
+- In cases of an error while reading the response body, the newly added `responseCallback` in v0.31.0 would be evaluated with the returned status code, while the reported one would be `0`, as the response errored out and k6 does not return incomplete responses. Now `responseCallback` will also receive a `0` status. ([#1962](https://github.com/k6io/k6/pull/1962))
+- Fix Kafka output not being usable with the InfluxDB format after v0.31.0 changes. [#1914](https://github.com/k6io/k6/pull/1914)
+- Error out with a user friendly message if `ramping-vus` executor would've not run a single iteration instead of just doing nothing. [#1942](https://github.com/k6io/k6/pull/1942)
+
+## Internals
+
+- JS: Added a way for JS modules to have per VU initialization and object. This can also be used by xk6 modules by implementing `github.com/loadimpact/k6/js/modules#HasModuleInstancePerVU`. (Part of [#1911](https://github.com/k6io/k6/pull/1911))
+
+
+## Breaking changes
+
+### Support for `ArrayBuffer` in all k6 JS APIs ([#1841](https://github.com/k6io/k6/pull/1841))
+
+Continuing from k6 [v0.31.0](https://github.com/k6io/k6/releases/tag/v0.31.0), we're finalizing the transition to `ArrayBuffer` values for working with binary data. This release introduces some changes that might break scripts that relied on the previous `array of integers` or `string` result types, returned by some of our JS APIs. Specifically these cases now return `ArrayBuffer` instead: `open(..., 'b')`, HTTP response bodies for requests that specified `responseType: 'binary'` (including when `http.batch()` is used), `crypto.randomBytes()`, `hasher.digest('binary')` and `encoding.b64decode()`.
+  The previous default behavior of returning string from `encoding.b64decode()` can be replicated with a new optional `format` argument and a value of `"s"`: `encoding.b64decode("5bCP6aO85by-Li4=", "url", "s")`.
+  Most of these shouldn't cause issues if the script is simply passing the values to another k6 API (e.g. opening a file as binary and passing it to `http.post()`), but in other cases the script will need to be modified to wrap the `ArrayBuffer` in a [typed array view](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Typed_arrays) and use that API instead.
+  
+
+### Deprecating official 32-bit binary releases ([#2000](https://github.com/k6io/k6/pull/2000))
+
+We have stopped offering official builds for 32-bit Windows and Linux binaries. 
+
+### Moved repo and renamed k6 Go module paths to `go.k6.io/k6` ([#2010](https://github.com/k6io/k6/pull/2010))
+
+We moved the repository location from https://github.com/loadimpact/k6 to https://github.com/k6io/k6. Instead of also moving the Go module paths to the new repo location, we decided to use custom ones with our own domain, for more control. This will be a breaking change for all [xk6](https://github.com/k6io/xk6) extensions, since they imports parts of k6 by source to register themselves.
```

Comment:
```suggestion
We moved the repository location from https://github.com/loadimpact/k6 to https://github.com/k6io/k6. Instead of also moving the Go module paths to the new repo location, we decided to use custom ones with our own domain, for more control. This will be a breaking change for all [xk6](https://github.com/k6io/xk6) extensions, since they import parts of k6 by source to register themselves.
```